doc/doc.go

```diff
@@ -117,9 +117,13 @@ func outputs(list *ast.ObjectList) []Output {
 		if is(item, "output") {
 			name, _ := strconv.Unquote(item.Keys[1].Token.Text)
 
+			items := item.Val.(*ast.ObjectType).List.Items
```

Comment:
I've not seen any panics thus far in my testing, and that's including modules with no outputs...